### PR TITLE
mrc-1867 add endpoint to fetch cases averted graph config

### PR DESCRIPTION
**Pull request by hillalex**
this PR adds the endpoint for the remaining graph: `/graph/impact/cases-averted/config`

### CODE REVIEW

File: inst/json/graph_cases_averted_config.json [hunk 1]
```diff
@@ -0,0 +1,121 @@
...
+        "title": "Clinical cases averted per 1,000 people per year",
+        "yaxis": {
+            "title": "cases averted",
+            "showline": true,
+            "autorange": true
+        },
+        "xaxis": {
+            "title": "intervention",
+            "showline": true,
+            "tickvals": ["ITN", "PBO", "IRS", "ITN + IRS", "PBO + IRS"],
```
Comment (richfitz) [inline inst/json/graph_cases_averted_config.json:115]: These intervention names look different to those used in the prevalence graph. We should standardise these somewhere as I feel we're getting a lot of duplication though the code and json here

**Review by richfitz — CHANGES_REQUESTED** [review]
This looks fine except I'm concerned about the incomplete duplication of intervention names. I wonder if we should centralise that all into a json file with the various abbreviations and full names, and reference that within the various schemas? Or in the short term at least be consistent or document being ok with not being consistent

**Comment by hillalex** [comment]
> This looks fine except I'm concerned about the incomplete duplication of intervention names. I wonder if we should centralise that all into a json file with the various abbreviations and full names, and reference that within the various schemas? Or in the short term at least be consistent or document being ok with not being consistent

Presuming the intervention names in the prev graph are taken directly from Ellie's data? I'll change this over and document them for now, agree would be nice to auto-populate them somehow. Or perhaps could just write unit tests that validate them against the master list...

**Comment by hillalex** [comment]
Sorry didn't mean to request review here - this should wait until https://github.com/mrc-ide/mintr/pull/18 is merged and then use the same approach for testing the config.

**Comment by hillalex** [comment]
Ok, this one now ready for review @richfitz

**Review by richfitz — APPROVED** [review]

**Pull request merged**